applications/openshift/kubelet/kubelet_configure_tls_cipher_suites/rule.yml

```diff
@@ -21,10 +21,18 @@ severity: medium
 references:
   cis@ocp4: 4.2.13
 
-ocil_clause: "TLS cipher suite configuration is not configured"
+ocil_clause: "TLS cipher suite configuration is not configured or contains insecure ciphers"
```

Comment:
Now that you're here, could you also add a description that indicates how to remediate this? in the `description` section.